draft-ietf-ppm-dap.md

```diff
@@ -1206,18 +1207,18 @@ This message consists of:
   Helper MUST abort with error "queryMismatch".
 
 * The sequence of report shares to aggregate. The `encrypted_input_share` field
-  of the report share is the `HpkeCiphertext` whose index in
-  `Report.encrypted_input_shares` is equal to the index of the aggregator in the
-  task's `aggregator_endpoints` to which the `AggregationJobInitReq` is being
-  sent.
+  of the `ReportAggregationInitReq` is the `HpkeCiphertext` whose index in
+  `Report.encrypted_input_shares` is equal to the index of the Helper, i.e. 1.
```

Comment:
I don't mind doing this, but it can be added on in a later PR -- let's get the technical bits correct here before we update all of the other text, to keep scope in check?